.travis.yml

```
@@ -2,3 +2,5 @@ language: node_js
 node_js:
   - 6
   - 8
+  - 10
+  - 12
```

Any reason to not include current?
```suggestion
  - 12
  - 14
```

---

TIL about v14!!

---

It's been current for a while. 12 went LTS in October:

 https://nodejs.org/en/about/releases/